packages/x-utils/tsconfig.esm.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "module": "esnext",
+    "declarationDir": null,
+    "sourceMap": true,
+    "inlineSources": true,
```

Comment:
`sourcemap` generation should be done for both builds, right? We should move this two properties to the common `tsconfig.json` file.